modules/ossm-add-project-using-label-selectors-cli.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-adding-project-using-label-selectors-cli_{context}"]
+= Adding a project to the {SMProductShortName} using label selectors with the CLI 
+
+You can use labels selectors to add a project to the {SMProductShortName} with the CLI.
+
+.Prerequisites
+* An installed, verified {SMProductName} Operator.
+* An existing `ServiceMeshMemberRoll` resource.
+* To perform this task, a user must have mesh admin privileges.
```

Comment:
```suggestion
* You are logged in as a user with mesh admin privileges.
```
- no need to say 'must'; it's implied by the fact that these are prereqs

Reply:
Done